sympy/logic/boolalg.py

```diff
@@ -186,6 +197,10 @@ class BooleanFalse(with_metaclass(Singleton, BooleanAtom)):
     False they act bitwise on 0. Functions in the logic module will return this
     class when they evaluate to false.
 
+    Notes
+    ======
+    See note in :py:class`sympy.logic.boolalg.BooleanTrue`
```

Comment:
A link should always indicate what kind of information can be found at the other end, otherweise the reader won't know if following the link is worth the context switch.